dev_docs/tutorials/adding_performance_metrics.mdx

```diff
@@ -159,6 +159,50 @@ to follow if it's important for you to look inside of a specific event e.g. `pag
 - **Keep performance in mind**. Reporting the performance of Kibana should never harm its own performance. 
   Avoid sending events too frequently (`onMouseMove`) or adding serialized JSON objects (whole `SavedObjects`) into the meta object.
 
+### Benchmarking performance on CI
+
+One of the use cases for event based telemetry is benchmarking the performance of features over time.
+In order to keep track of their stability, the #kibana-operations team has developed a special set of
+functional tests called `Journeys`. These journeys execute a UI workflow and allow the telemetry to be
+reported to a cluster where it can then be analysed.
+
+Those journeys run on the key branches (main, release versions) on dedicated machines to produce results 
+as stable and reproducible as possible.
+
+#### Machine specifications
+
+The machine has these [specs](https://www.hetzner.com/dedicated-rootserver/ex100).
```

Comment:
I think it's ok for now but Hetzner tends to retire instance types from time to time which will break the link. Also, we might choose different instance types and this information gets outdated then. I suggest:

* Write down the machine specs (CPU, memory, disk) in addition to the link to Hetzner
* Should we one day have a dedicated page for benchmarks, we can document this information there.

You can see an example for both suggestions in https://elasticsearch-benchmarks.elastic.co/ (in the section `Benchmarking Environment`).